### ISSUE TITLE
Users can not create alert monitors with index per namespace enabled

### ISSUE THREAD

**Issue by davidumea** [elastisys/compliantkubernetes-apps#1557]
**Describe the bug**
Users mapped to `alerting_full_access` can not create alert monitors when index per namespace is enabled.

The users can not select an index in the "data source" tab.

These are the required index permissions:
```
indices_monitor
indices:admin/aliases/get
indices:admin/mappings/get
```

Problem is that when you grant those permissions to these indices ([which we use for other roles](https://github.com/elastisys/compliantkubernetes-apps/blob/main/helmfile/values/opensearch/configurer.yaml.gotmpl#L125-L128)):
```
/^(?!kube-system|gatekeeper-system|other|authlog|\.).*/
```
the users are still not able to pick any index as datasource when creating an alerting monitor.

It only works if you grant them those permissions on all indices, `*`.

**To Reproduce**
1. Enable index per namespace
2. Log in to Opensearch as a user mapped to roles `kibana_user`, `kubernetes_log_reader` & `alerting_full_access`. (Most common non-cluster-admin permissions)
3. Go to Alerting -> Monitors -> Click "Create monitor"
4. Try to pick an index (or index pattern) as a datasource.

**Expected behavior**
User with `alerting_full_access` should be albe to create alerting monitors, with index per namespace enabled.

**Version (add all relevant versions):**
 - Compliant kubernetes apps version [v0.29.1]

**Comment by lunkan93** [comment]
Waiting for reply to [issue upstream](https://github.com/opensearch-project/alerting/issues/926)

Issue seems to occur with or without index per namespace and no matter what you specify other than `"*"`.

**Comment by lunkan93** [comment]
Checked back in on this when upgrading the chart - Seems that you can search for indices and select them that way, but the drop-down menu still isn't working correctly.

![Screenshot from 2024-03-18 12-46-41](https://github.com/elastisys/compliantkubernetes-apps/assets/60571221/9280de47-ae19-4666-9e7b-8d2393637d35)

**Comment by OlleLarsson** [comment]
@Ajarmar add bug note in internal docs?

**Comment by salehsedghpour** [comment]
Can we have the bug note in our internal docs @lunkan93 ?

**Comment by davidumea** [comment]
This can be closed right @lunkan93

**Comment by lunkan93** [comment]
Yeah I would say so, drop-down probably wont be fixed but the feature should work other than that.
